docs/manual/en/introduction/Libraries-and-Plugins.html

```diff
@@ -111,6 +111,7 @@ <h3>Wrappers and Frameworks</h3>
 			<li>[link:https://threlte.xyz/ Threlte] - Svelte components for 3D graphics built on Three.</li>
 			<li>[link:https://needle.tools/ Needle Engine]</li>
 			<li>[link:https://tresjs.org/ tresjs] - Vue components for 3D graphics built on Three.</li>
+			<li>[link:https://giro3d.org Giro3D] - Versatile framework built on Three for visualizing and interacting with Geospatial 2D, 2.5D and 3D data.</li>
```

Comment:
Do you mind adding this link to all translated pages as well?

You don't have to actually translate the text, just adding the same markup like in the `en` version is fine.